apps/dashboard/app/helpers/scripts_helper.rb

```diff
@@ -60,4 +60,8 @@ def auto_accounts_template
   def script_form_double
     BootstrapForm::FormBuilder.new('script', nil, self, {})
   end
+
+  def script_remove_field_enabled(id)
+    !['script_auto_scripts'].include?(id.to_s)
+  end
```

Comment:
Can you change this to a question like `removable?` or `removable_field?`

Reply:
updated